thunder/torch/experimental/dtensor_torch_and_prims.py

```diff
@@ -134,5 +135,60 @@ def dtensor_mul(a: TensorLike, b: TensorLike) -> TensorLike:
     return dtensor_mul_prim(a, b)
 
 
+def dtensor_convert_element_type_meta(a, dtype):
+    tdtype = ltorch.to_torch_dtype(dtype)
+    output = run_with_fake_tensor(lambda x, dt: x.to(dt), a, tdtype)
+    local_tensor_proxy = TensorProxy(like=a.local_tensor, shape=output._local_tensor.shape, dtype=dtype)
```

Comment:
Local tensor could be obtained by running prims.convert_element_type.

Why is it better to run with fake tensors? What happens if the shape returned by fake tensor ops is different from prims.convert_element_type?